src/sidekick/module.js

```diff
@@ -883,6 +883,21 @@
     }
   }
 
+  /**
+   * Check if current sidekick is on a supported Google Doc based on contentType.
+   * @private
+   * @param {String} contentType
+   */
+  function isSupportedGoogleType(contentType) {
+    const types = ['application/vnd.google-apps.document', 'application/vnd.google-apps.spreadsheet'];
+    for (const itype of types) {
+      if (itype === contentType) {
+        return true;
+      }
+    }
+    return false;
+  }
```

Comment:
this is a bit complicated... :-)

```suggestion
  function isSupportedGoogleType(contentType) {
    return contentType in {
      'application/vnd.google-apps.document': true,
      'application/vnd.google-apps.spreadsheet': true,
    };
  }
```

or even just:

```suggestion
  function isSupportedGoogleType(contentType) {
    return contentType === 'application/vnd.google-apps.document'
      || contentType === 'application/vnd.google-apps.spreadsheet';
  }
```

Reply:
@tripodsan Made the simple change. Trying to figure out how these tests work. :) Thank you for reviewing.